cli/main.go

```diff
@@ -401,6 +412,11 @@ func main() {
 						out = &outProp
 					}
 
+					if simulateTransaction && len(sender) == 0 {
+						core.Panic("if using --print-calldata, please specify a --sender. Note that the transaction will not be broadcast.")
+						return nil
+					}
```

Comment:
Why do they need to specify a sender? Is it so we can simulate the txn for them?

If so, we can query their pod's `podOwner` address and use that as the sender. They shouldn't need to supply this.

Reply:
solved this by just hardcoding a random address. the signed data isn't used, but to simulate the txn (so that we can get the calldata in a strongly typed way) there needs to be SOME signer.

abigen isn't great :(